pandas/core/generic.py

```diff
@@ -5201,60 +5201,209 @@ def abs(self):
         """
         return np.abs(self)
 
```

Comment:
why are you redefining things here???

this is just a very small edit to the ``_shared_docs['describe']``